eng/testing/tests.wasm.targets

```diff
@@ -154,13 +154,13 @@
     <WorkloadIdForTesting Include="wasm-tools-net7;wasm-experimental-net7"
                           ManifestName="Microsoft.NET.Workload.Mono.ToolChain.net7"
                           Variant="net7"
-                          Version="$(PackageVersion)"
+                          Version="$(PackageVersionForWorkloadManifests)"
```

Comment:
If this is a required property to be set, consider adding a target that validates it and otherwise throw via the Error msbuild task.

Reply:
If this isn't set correctly then it will fail when we try to install the workload for testing. And that emits better debug info for it too.